Source/Fuse.Common/Internal/iOSSystemFont.uno

```diff
@@ -51,18 +69,71 @@ namespace Fuse.Internal
 
 		public static List<FontFaceDescriptor> Get(string family, Fuse.SystemFont.Style style, Fuse.SystemFont.Weight weight)
 		{
-			var weightIndex = Math.Clamp((int)weight, 0, _weightNames.Length - 1);
-			var descriptor = GetMatchingFontDescriptor(family, style == Fuse.SystemFont.Style.Italic, _weightNames[weightIndex]);
-			if (descriptor == null)
-			{
-				return Default;
-			}
-			else
-			{
+			#if defined(__IPHONE_13_0) && __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_13_0
+			if (Fuse.iOSDevice.OperatingSystemVersion.Major >= 13) { //system font for iOS 13
+
+				//satisfy custom font structure
+				var path = "/System/Library/Fonts/Core/AppleSystemUIFont-" + style + weight;
+				var fontName = ".AppleSystemUIFont-" + style + "-" + weight;
+
+				var ffd = new FontFaceDescriptor(new SystemFileSource(path), GetFontNameStyles(fontName));
+				ffd.SetFontAttributes(style,weight,Fuse.SystemFont.Design.Default, 16, false);
+				var result = new List<FontFaceDescriptor>();
+				result.Add(ffd);
+				return result;
+			} else { //normal custom font
+			#endif
+
+				var weightIndex = Math.Clamp((int)weight, 0, _weightNames.Length - 1);
+				var descriptor = GetMatchingFontDescriptor(family, style == Fuse.SystemFont.Style.Italic, _weightNames[weightIndex]);
+				if (descriptor == null)
+				{
+					descriptor = GetDefaultFontDescriptor(_weightNames[weightIndex], (style == Fuse.SystemFont.Style.Italic));
+				}
+				descriptor = GetFallbackUIFontDescriptorsWeight(descriptor, family, _weightNames[weightIndex], (style == Fuse.SystemFont.Style.Italic));
 				var descriptors = GetFallbackUIFontDescriptors(descriptor);
 				var fontNames = GetFontNamesFromUIFontDescriptors(descriptors);
+
 				return GetDescriptorsFromFontNames(fontNames);
+
+			#if defined(__IPHONE_13_0) && __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_13_0
 			}
+			#endif
+		}
+
+		static ObjC.Object GetFallbackUIFontDescriptorsWeight(ObjC.Object descriptor, string family, string weightName, bool isItalic) 
+		{
+			var fontName = (!isItalic) ? GetDescriptorFontName(descriptor) : GetDescriptorName(descriptor);
+			if (weightName == "UltraLight" && isItalic == false && (DoesFontWeightExist(family, fontName, weightName) == false)) {
```

Comment:
Curly braces should go on separate lines, but in this case all curly braces can be removed because it's all single statements.